docs/data/osdf/install-origin-container.md

```diff
@@ -0,0 +1,353 @@
+| `ISSUER_JWK`          | `/etc/osdf-origin/issuer.jwk`    | The private key generated by `pelican generate keygen`                                  | 
+| `HOST_CERT_CHAIN`     | `/etc/osdf-origin/hostcert.pem`  | The host certificate, with full chain                                                   | 
+| `HOST_KEY`            | `/etc/osdf-origin/hostkey.pem`   | The private key matching the host certificate                                           | 
+| `ORIGIN_CONFIG_FILE`  | `/etc/osdf-origin/config.yaml`   | The YAML file containing various origin settings                                        | 
+| `OUTSIDE_DIRECTORY`   | `/mnt/origin`                    | The directory outside the container that stores the data to be served                   | 
+| `INSIDE_DIRECTORY`    | `/pelican`                       | The path inside the container corresponding to StoragePrefix in your origin config file |
+
+
+```console
+user@host $ docker run --rm \
+             --publish <ORIGIN_HOST_PORT>:8443 \
+             --publish <WEB_INTERFACE_HOST PORT>:8444 \
+             --volume <ISSUER_JWK>:/etc/pelican/issuer.jwk \
+             --volume <HOST_CERT_CHAIN>:/etc/pelican/certificates/tls.crt \
+             --volume <HOST_KEY>:/etc/pelican/certificates/tls.key \
+             --volume <ORIGIN_CONFIG_FILE>:/etc/pelican/config.d/99-local.yaml \
+             --volume <OUTSIDE_DIRECTORY>:<INSIDE_DIRECTORY> \
+             --name osdf-origin \
+             hub.opensciencegrid.org/pelican_platform/osdf-origin:latest
+```
+
+Using the example values from the table, this is
+
+```console
+user@host $ docker run --rm \
+             --publish 8443:8443 \
+             --publish 8444:8444 \
+             --volume /etc/osdf-origin/issuer.jwk:/etc/pelican/issuer.jwk \
+             --volume /etc/osdf-origin/hostcert.pem:/etc/pelican/certificates/tls.crt \
+             --volume /etc/osdf-origin/hostkey.pem:/etc/pelican/certificates/tls.key \
+             --volume /etc/osdf-origin/config.yaml:/etc/pelican/config.d/99-local.yaml \
+             --volume /mnt/origin:/pelican \
+             --name osdf-origin \
+             hub.opensciencegrid.org/pelican_platform/osdf-origin:latest
+```
+
+
+Validating the Origin
+---------------------
+
+Download a test file (POSIX) or object (S3) from your origin (replacing `ORIGIN_HOSTNAME` with the host name of your origin,
+and TEST_PATH with the OSDF path to the test file or object)
+
+```
+user@host$ curl -L https://ORIGIN_HOSTNAME:8443/TEST_PATH -o /tmp/testfile
+```
+
+Verify the contents of `/tmp/testfile` match the test file or object your origin was serving.
+
+Debugging information will be in your container logs e.g., `docker logs osdf-origin`.
+To increase the debugging information in the origin, edit your origin configuration file and set:
+```
+Debug: true
+```
```

Comment:
I know we've talked about it before but I just want to note that this assumes a public namespace. We don't have to hold up the PR for this but in the future, I think we should:

1. Tell people to use the `pelican` client. The `--debug` info is nice for troubleshooting, it's easier to specify a token, etc
2. Tell people to set `Logging.Origin.Scitokens: trace` if they run into problems

Reply:
Done